src/react/components/pages/editorPage/assetPreview.tsx

```diff
@@ -13,6 +13,7 @@ export interface IAssetProps {
     onActivated?: (contentSource: ContentSource) => void;
     onDeactivated?: (contentSource: ContentSource) => void;
     onChildAssetSelected?: (asset: IAsset) => void;
+    framerate?: number;
```

Comment:
Instead of passing only the framerate - let's wrap this up into some larger options object that we can then pass that is derived from the project image/video settings.